src/components/GeneralCharts/ChoicesCharts.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+} from 'chart.js';
+import { Bar } from 'react-chartjs-2';
+import ChartDataLabels from 'chartjs-plugin-datalabels';
+
+ChartJS.register(
+  ChartDataLabels,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend
+);
+
+export const ChoicesCharts = ({ gameChoiceValues, userChoices }) => {
+  function getGameChoicesDescription() {
+    let output = { description: [], numChoice: [] };
+
+    for (let i = 0; i < gameChoiceValues.length; i++) {
+      const maxCharacterCount = Math.round(1000 / gameChoiceValues.length / 16);
+      let gameChoiceValuesDescription = gameChoiceValues[i].description;
+      if (gameChoiceValuesDescription.length > maxCharacterCount) {
+        let descriptionSplit = [];
+        let end = maxCharacterCount;
+        let start = 0;
+        while (end < gameChoiceValuesDescription.length) {
+          let hasDash = false;
+          if (
+            gameChoiceValuesDescription[end - 1] !== ' ' &&
+            gameChoiceValuesDescription[end] !== ' ' &&
+            end < gameChoiceValuesDescription.length
+          ) {
+            hasDash = true;
+          }
+          descriptionSplit.push(
+            gameChoiceValuesDescription.slice(start, end) +
+              `${hasDash ? '-' : ''}`
```

Comment:
Just wondering if there's any way around the description having dashes when we mouse-over the bar?

<img width="405" alt="Screenshot 2022-05-07 at 12 42 26 PM" src="https://user-images.githubusercontent.com/41856541/167238631-37e68e91-171e-478d-bce8-f0a83d197d2e.png">

Reply:
Hmmmmmmmm maybe we can forego the tooltip since we ald have all the data and description shown on screen

Reply:
Fair point. The tooltip doesn't add any additional info at all